lib/main_publiccloud.pm

```diff
@@ -160,6 +162,7 @@ sub load_latest_publiccloud_tests {
     }
     elsif (get_var('PUBLIC_CLOUD_UPLOAD_IMG')) {
         loadtest "publiccloud/upload_image", run_args => $args;
+        loadtest "publiccloud/azure_aitl", run_args => $args;
```

Comment:
this line needs to be removed , I assume you simply forgot to tweak this part ...

Reply:
Indeed. The whole test needs to  be moved elsewhere, I forgot to move it.